doc/source/whatsnew/v0.23.2.txt

```diff
@@ -16,8 +16,12 @@ and bug fixes. We recommend that all users upgrade to this version.
 Fixed Regressions
 ~~~~~~~~~~~~~~~~~
 
+- 
 -
--
+
+**Other Fixes**
+
+- Bug in :meth:`first_valid_index` that raised for row index with duplicate values (:issue:`21441`)
```

Comment:
I think this should be :meth:\`DataFrame.first_valid_index\`

Reply:
you don't need a separate sub-section here, just list the issue

'raised for a row index'

Reply:
Thanks - updated
Have left it as :meth:`first_valid_index` as this issue affects both DataFrame and Series (though the example and title of the original issue points only to DataFrame)